calcom/cal.com
# fix "copy time to" issue when deselecting a day

## What does this PR do?

Fixes #8805

**Environment**:  Production

## Type of change

Bug Fix
Adds a fix  when you use multiple days to copy the time to, then remove one of those days and click the apply button. It only copy time to the removed day.

## How should this be tested?

Try copying to multiple days and deselect one of the chosen days and see if it works.

https://www.loom.com/share/f3263725fbc246488dd948714751e49b
